mlrun/__main__.py

```diff
@@ -1045,7 +1051,13 @@ def project(
         proj.spec.params["commit_id"] = commit
     if secrets:
         secrets = line2keylist(secrets, "kind", "source")
-        proj._secrets = SecretsStore.from_list(secrets)
+        secret_store = SecretsStore.from_list(secrets)
+        # Allow the user to run a workflow with secrets on runtime, without using or storing k8s secrets.
+        # if the user wants to run a schedule workflow or to use those secrets in other runs he will need to save
+        # those secrets in k8s and use the --save-secret flag
```

Comment:
Used to run a workflow with secrets in runtime, without using or storing k8s secrets.
To run a scheduled workflow or to use those secrets in other runs, save
the secrets in k8s and use the --save-secret flag

Reply:
Fix, thanks